orcid_service/views.py

```diff
@@ -352,6 +351,17 @@ def update_status(orcid_id):
             else:
                 bibcodes = payload['bibcodes']
             profile.update_status(bibcodes,payload['status'])
+
+            session.begin_nested()
+            try:
+                session.add(profile)
+                session.commit()
+            except exc.IntegrityError:
+                session.rollback()
+                return json.dumps({'msg': 'Updated status for bibcode {} not saved'.format(bibcodes)}), 500
+            # per PEP-0249 a transaction is always in progress
+            session.commit()
```

Comment:
Do we need this second commit? This will only get executed if the first commit succeeds.